Test/TestRetentionTimePrediction.cs

```diff
@@ -643,6 +642,15 @@ public void SSRCalc3_100A_Test()
                 }
             }
         }
+
+        /// <summary>
+        ///A test for CZE retention time prediction
+        ///</summary>
+        [Test]
+        public void CZE_RetentionTime_Test()
+        {
```

Comment:
There's already a test for this... https://github.com/smith-chem-wisc/mzLib/blob/master/Test/TestCZE.cs

Reply:
I was going for a more comprehensive test, since the current one has a 26% coverage...
![image](https://user-images.githubusercontent.com/16883585/45051123-bf64fa80-b048-11e8-8b3e-30cdb48619cc.png)

Reply:
Cool, just didn't want you to spend time writing the same thing.